### PR TITLE
Create Layout-ResponsiveDesign.md

### PR DESCRIPTION
**Pull request by AmrutaJayanti**
## Related Issue

Fix: #540 

## Description

Created `Layout-ResponsiveDesign.md`

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [x] Documentation update
- [ ] Other (specify): _______________

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: docs/CSS/css-basics/Layout-ResponsiveDesign.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Layout in CSS
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-basics/Layout-ResponsiveDesign.md:1]: You want to need on to

```md
---
id: _____
title: ____
sidebar_label: ___
sidebar_position: ____
tags: _____
description: _____
---
```

for example

```md
---
id: intro-HTML
title: Introduction of HTML
sidebar_label: Introduction of HTML
sidebar_position: 1
tags: [html, introduction, web development, markup language, hyper text, web pages, career opportunities, personal growth, web-development, web design, web pages, websites, career opportunities, contribute to the web, stay relevant, express yourself, learn other technologies, have fun, how to use html, steps to start using html, set up your development environment, create your first html document, learn html syntax and structure, explore html elements and-attributes]
description: In this tutorial, you will learn about HTML, its importance, what is HTML, why learn HTML, how to use HTML, steps to start using HTML, and more.
---
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
now, update your code based on suggestions

and also, use 

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       <!-- your html / xml / jsx code here -->
 </BrowserWindow>
```

for example

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
       <div>
         <h1>This is a Heading</h1>
         <p>This is a paragraph.</p>
       </div>
     </BrowserWindow>
```

and live preview

image

**Comment by AmrutaJayanti** [comment]
@Ajay-Dhangar I made changes.Please approve

File: docs/CSS/css-basics/Layout-ResponsiveDesign.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+2. *Positioning:* CSS positioning allows you to control the position of elements on a web page. There are different types of positioning:
+
+   - `static`: This is the default positioning. Elements are positioned according to the normal flow of the document.
+   - `relative`: Elements are positioned relative to their normal position in the document flow.
+   - `absolute`: Elements are positioned relative to their nearest positioned ancestor, or the containing block.
+   - `fixed`: Elements are positioned relative to the viewport, so they remain in the same position even when the page is scrolled.
+   - `sticky`: Elements are positioned relative to the viewport like fixed positioning, but they behave like relative positioning until they reach a specified scroll position.
+
+<BrowserWindow url="http://127.0.0.1:5500/index.html">
+<div class="relative" style="position: relative;
```
Comment (Ajay-Dhangar) [inline docs/CSS/css-basics/Layout-ResponsiveDesign.md]: follow jsx syntax like 

```md
<BrowserWindow url="http://127.0.0.1:5500/index.html">
      <div
        style={{
          position: 'relative',
          top: '20px',
          left: '30px',
        }}
      >
        Relative positioning
      </div>
      <div
        style={{
          position: 'absolute',
          top: '50px',
          left: '100px',
        }}
      >
        Absolute positioning
      </div>
</BrowserWindow>
```

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**